plugin/accel_embree/accel_embree.cpp

```diff
@@ -33,9 +33,28 @@ class Accel_Embree final : public Accel {
 private:
     RTCDevice device_ = nullptr;
     RTCScene scene_ = nullptr;
+    RTCBuildArguments settings_;
+    RTCSceneFlags sf_;
     std::vector<FlattenedPrimitiveNode> flattened_nodes_;
 
 public:
+
+     virtual void construct(const Json& prop) override {
+        settings_ = rtcDefaultBuildArguments();
+        from_json(prop,settings_);
+
+        from_json(prop,sf_);
+
+        Json j;
+        to_json(j,settings_);
+        LM_INFO(j.dump());
+        j.clear();
+        
+        to_json(j,sf_);
+        LM_INFO( j.dump() );
+        
+        }
+
```

Comment:
to_json and from_json functions can be implemented with nlohman_json's adl_serialier. ref: `https://github.com/nlohmann/json`